src/poliastro/plotting.py

```diff
@@ -3,22 +3,21 @@
 """
 import os.path
 from itertools import cycle
-
-import numpy as np
```

Comment:
Can you keep the imports as they are to reduce the number of changed lines? This will reduce the noise for the reviewer :)

Reply:
Okay! Working on that!

Reply:
I used isort tool to resolve the problems. I guess that shifted the line below others.